js/index.js

```diff
@@ -323,3 +323,18 @@ window.addEventListener('load', () => {
     fullBody.classList.toggle('active');
   });
 });
+
+const form = document.getElementById('contact-information');
+
```

Comment:
[OPTIONAL] When I submit the form, the error message shows even if the email is in lowercase. Kindly look into this and make sure that it only shows when the email contains uppercase letters.

![image](https://user-images.githubusercontent.com/38088818/174392097-d973d99c-7c24-49df-8f6d-c65843c7cc6b.png)

Reply:
hi 
thanks for the great feedback 
yeah i removed it 
i used (remove) instead of (toggle)

Reply:
Fantastic job!!! 👍🏾